tests/E2E/EntryTest.php

```diff
@@ -155,8 +155,8 @@ public function testAssetsResolvedFromIncludes()
         $this->assertSame('nyancat', $image->getId());
 
         // There should be 3 and only 3 requests:
```

Comment:
4 and only 4?

Reply:
Oh yeah, I forgot to update the comment 😁